hack/common.sh

```diff
@@ -4,10 +4,9 @@ function pack_build() {
     image=$1
     target=$2
     pack_args=${@:3}
-    builder="gcr.io/cf-build-service-public/ci/kpack-builder" # builder used ci
-
-    pack build ${image} --builder ${builder} -e BP_GO_TARGETS=${target} ${pack_args} --publish --trust-builder
-
+    (cd hack && docker build -t kpack-builder .)
+    builder="kpack-builder" # builder used ci
+    pack build ${image} --builder ${builder} -e BP_GO_TARGETS=${target} ${pack_args} --publish --trust-builder -t ${image}:${kpack_version} --cache-image ${image}-cache
```

Comment:
Added a cache image flag here to speed up GH rebuilds. I also had to add an extra tag here so that we can get proper versioning on the output image and so that github doesn't GC the untagged releases once latest has been pushed.